test/t8_cmesh/t8_gtest_cmesh_copy.cxx

```diff
@@ -34,17 +34,18 @@
  * We test whether the new and original cmesh are equal.
  */
 
-/* Note: This test currently fails on many cmeshes and is thus deavtivated.
- * See: https://github.com/DLR-AMR/t8code/issues/920
- */
-
-/* Remove `DISABLED_` from the name of the Test(suite) or use `--gtest_also_run_disabled_tests` when you start working on the issue. */
-class DISABLED_t8_cmesh_copy: public testing::TestWithParam<cmesh_example_base *> {
+class t8_cmesh_copy: public testing::TestWithParam<cmesh_example_base *> {
  protected:
   void
   SetUp () override
   {
-    /* Skip test since cmesh copy is not yet working. See https://github.com/DLR-AMR/t8code/issues/920 */
+    if (GetParam ()->name == std::string { "t8_cmesh_new_disjoint_brick_" }) {
+      GTEST_SKIP ();
+    }
+    else {
```

Comment:
We can delete the writing of the testname, as GTest writes the testname at the beginning of the testsummary automatically.